jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -4,172 +4,199 @@
 
 /** Status React component used to display Juju status. */
 class Status extends React.Component {
+  /**
+    Return an element class name suitable for the given value.
+    @param {String} value The provided value.
+    @returns {String} The class name ('ok', 'error' or '').
+  */
+  _getClass(value) {
+    switch (value) {
+      case 'active':
+      case 'idle':
+      case 'started':
+        return 'ok';
+      case 'blocked':
+      case 'down':
+      case 'error':
+        return 'error';
+    }
+    return '';
+  }
 
   /**
-    Render the component.
+    Sort by the given index.
+    @param index {Int} The index to sort by.
+    @param {Object} a The first value.
+    @param {Object} b The second value.
+    @returns {Array} The sorted array.
   */
-  render() {
-    return (
-      <juju.components.Panel
-        instanceName="status-view"
-        visible={true}>
-        <div className="status-view__content">
-          {this.renderStatus()}
-        </div>
-      </juju.components.Panel>
-    );
+  _byIndex(index, a, b) {
+    if (a[index] < b[index])
+      return -1;
+    if (a[index] > b[index])
+      return 1;
+    return 0;
   }
 
   /**
-    Render the current model status.
```

Comment:
We really don't use tables for anything, it is rarely used at all in HTML these days (see my explanation above).